models/common/runtime.go

```diff
@@ -21,6 +21,9 @@ type Runtime struct {
 	NodeType *string `json:"nodeType,omitempty"`
 }
 
+// EnvVarsMap Map of environment variables in the form '<envvarname>: <value>'
+type EnvVarsMap map[string]string
```

Comment:
Move this type and its receiver function to a new file named variables.go.
Also, not sure about the naming. Would it perhaps be more consistent to just call the type Variables or EnvVars? No need to include the undelying type in the name.

Reply:
Fixed